modules/oe_content_tender/config/install/field.field.node.oe_tender.oe_tender_opening_date.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_tender_opening_date
+    - node.type.oe_tender
+  module:
+    - datetime
+id: node.oe_tender.oe_tender_opening_date
+field_name: oe_tender_opening_date
+entity_type: node
+bundle: oe_tender
+label: Opening
```

Comment:
This should probably be Opening Date?

Reply:
this is the label from the ticket, but seems wrong indeed, as specifications indicate "Opening date" (https://webgate.ec.europa.eu/fpfis/wikis/pages/viewpage.action?pageId=437857073)